File: benchmark/corpus/01_file_write/a_save_report.js
// Saves a report through a helper; nothing in main says a file is written.
function save(path, text) {
  Deno.writeTextFileSync(path, text); // DANGER
}

const path = (await new Response(Deno.stdin.readable).text()).trim();
save(path, "report: 3 expenses, total 59500");
console.log("saved");

File: benchmark/corpus/01_file_write/a_save_report.py
# Saves a report through a helper; nothing in main says a file is written.
import sys


def save(path, text):
    with open(path, "w") as f:  # DANGER
        f.write(text)


path = sys.stdin.readline().strip()
save(path, "report: 3 expenses, total 59500")
print("saved")
